rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1278,6 +1278,42 @@ JobIdData submit(@HeaderParam("sessionid") String sessionId, @PathParam("path")
             MultipartFormDataInput multipart, @Context UriInfo contextInfos) throws JobCreationRestException,
             NotConnectedRestException, PermissionRestException, SubmissionClosedRestException, IOException;
 
+    /**
+     * Submits a job to the scheduler
+     *
+     * @param sessionId
+     *            a valid session id
+     * @param jobId
+     *            job id of the already submitted job
+     * @param pathSegment
+     *            variables of the workflow
+     * @return the <code>jobid</code> of the newly created job
+     * @throws PermissionRestException if user does not have rights to access job with <code>jobId</code>
+     */
+    @GET
+    @Path("jobs/{jobid}/reSubmit")
+    @Produces("application/json")
+    JobIdData reSubmit(@HeaderParam("sessionid") String sessionId, @PathParam("jobid")
+    final String jobId, @PathParam("path") PathSegment pathSegment) throws JobCreationRestException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)